backends/tfhe-cuda-backend/src/bindings.rs

```diff
@@ -0,0 +1,1684 @@
+pub type __u_short = ffi::c_ushort;
+pub type __u_int = ffi::c_uint;
+pub type __u_long = ffi::c_ulong;
+pub type __int8_t = ffi::c_schar;
+pub type __uint8_t = ffi::c_uchar;
+pub type __int16_t = ffi::c_short;
+pub type __uint16_t = ffi::c_ushort;
+pub type __int32_t = ffi::c_int;
+pub type __uint32_t = ffi::c_uint;
+pub type __int64_t = ffi::c_long;
+pub type __uint64_t = ffi::c_ulong;
+pub type __int_least8_t = __int8_t;
+pub type __uint_least8_t = __uint8_t;
+pub type __int_least16_t = __int16_t;
+pub type __uint_least16_t = __uint16_t;
+pub type __int_least32_t = __int32_t;
+pub type __uint_least32_t = __uint32_t;
+pub type __int_least64_t = __int64_t;
+pub type __uint_least64_t = __uint64_t;
+pub type __quad_t = ffi::c_long;
+pub type __u_quad_t = ffi::c_ulong;
+pub type __intmax_t = ffi::c_long;
+pub type __uintmax_t = ffi::c_ulong;
+pub type __dev_t = ffi::c_ulong;
+pub type __uid_t = ffi::c_uint;
+pub type __gid_t = ffi::c_uint;
+pub type __ino_t = ffi::c_ulong;
+pub type __ino64_t = ffi::c_ulong;
+pub type __mode_t = ffi::c_uint;
+pub type __nlink_t = ffi::c_ulong;
+pub type __off_t = ffi::c_long;
+pub type __off64_t = ffi::c_long;
+pub type __pid_t = ffi::c_int;
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct __fsid_t {
+    pub __val: [ffi::c_int; 2usize],
+}
+#[allow(clippy::unnecessary_operation, clippy::identity_op)]
+const _: () = {
+    ["Size of __fsid_t"][::std::mem::size_of::<__fsid_t>() - 8usize];
+    ["Alignment of __fsid_t"][::std::mem::align_of::<__fsid_t>() - 4usize];
+    ["Offset of field: __fsid_t::__val"][::std::mem::offset_of!(__fsid_t, __val) - 0usize];
+};
+pub type __clock_t = ffi::c_long;
+pub type __rlim_t = ffi::c_ulong;
+pub type __rlim64_t = ffi::c_ulong;
+pub type __id_t = ffi::c_uint;
+pub type __time_t = ffi::c_long;
+pub type __useconds_t = ffi::c_uint;
+pub type __suseconds_t = ffi::c_long;
+pub type __suseconds64_t = ffi::c_long;
+pub type __daddr_t = ffi::c_int;
+pub type __key_t = ffi::c_int;
+pub type __clockid_t = ffi::c_int;
+pub type __timer_t = *mut ffi::c_void;
+pub type __blksize_t = ffi::c_long;
+pub type __blkcnt_t = ffi::c_long;
+pub type __blkcnt64_t = ffi::c_long;
+pub type __fsblkcnt_t = ffi::c_ulong;
+pub type __fsblkcnt64_t = ffi::c_ulong;
+pub type __fsfilcnt_t = ffi::c_ulong;
+pub type __fsfilcnt64_t = ffi::c_ulong;
+pub type __fsword_t = ffi::c_long;
+pub type __ssize_t = ffi::c_long;
+pub type __syscall_slong_t = ffi::c_long;
+pub type __syscall_ulong_t = ffi::c_ulong;
+pub type __loff_t = __off64_t;
+pub type __caddr_t = *mut ffi::c_char;
+pub type __intptr_t = ffi::c_long;
+pub type __socklen_t = ffi::c_uint;
+pub type __sig_atomic_t = ffi::c_int;
+pub type int_least8_t = __int_least8_t;
+pub type int_least16_t = __int_least16_t;
+pub type int_least32_t = __int_least32_t;
+pub type int_least64_t = __int_least64_t;
+pub type uint_least8_t = __uint_least8_t;
+pub type uint_least16_t = __uint_least16_t;
+pub type uint_least32_t = __uint_least32_t;
```

Comment:
all those defs look useless ? 🤔 is this a known "issue" from bindgen ?

Reply:
I can prevent them from being generated but I have to specify for each of those in the build.rs with a configuration to bindgen:
```
.blocklist_type("__uint_least32_t")
```
I don't think I want to do this :sweat_smile:

Reply:
ok, could be worth investigating at some point to have less clutter in that file :)